guides/common/modules/con_managing-ostree-content.adoc

```diff
@@ -0,0 +1,20 @@
+[id="con_managing-ostree-content_{context}"]
+= Managing OSTree Content
+
+*OSTree* is a tool to manage bootable, immutable, versioned file system trees.
+
+OSTree makes it easy to install and update Linux-based operating systems on hosts and to switch between versions of operating systems on hosts.
+If a host update fails, you can easily revert the operating system on the host to the previous working version.
+
+You can create an OSTree image using an image builder and expose it in an OSTree repository on an HTTP server.
+Then you can use your {Project} to synchronize and manage OSTree branches from the exposed OSTree repository.
+
+.Prerequisites
+* In {ProjectServer}, OSTree management tools are enabled by default.
+If you ever have a reason to enable the tool, enter the following command:
```

Comment:
You might mention that it is el8 only.  (doesn't work on el7)

Reply:
Thanks for catching that! I intended, but forgot :)